tango_simlib/tango_sim_generator.py

```diff
@@ -401,16 +401,15 @@ def configure_device_model(sim_data_file=None, test_device_name=None):
     # In case there are more than one data description files to be used to configure the
     # device.
     parsers = []
-    for file_descriptor in data_file:
-        parsers.append(get_parser_instance(file_descriptor))
+    for file_name in data_file:
+        parsers.append(get_parser_instance(file_name))
 
     # In case there is more than one parser instance for each file
     model = Model(dev_name)
     for parser in parsers:
-        model_quantity_populator = PopulateModelQuantities(parser, dev_name, model)
-        sim_model = model_quantity_populator.sim_model
-        PopulateModelActions(parser, dev_name, sim_model)
-        PopulateModelProperties(parser, dev_name, sim_model)
+        PopulateModelQuantities(parser, dev_name, model)
+        PopulateModelProperties(parser, dev_name, model)
+    PopulateModelActions(parsers, dev_name, model)
```

Comment:
I thought the general problem of having definitions split over multiple input files would affect quantities and properties too, but now I see that `PopulateModelQuantities` and `PopulateModelProperties` already handle that case, so they need to stay in the loop.  It does make the `PopulateModel...` `__init__` method signatures slightly different (one takes a list, while the other two take a single instance).  Probably OK for now.

Reply:
Yes, that's right. But since `PopulateModelQuantities` reports on the file name if there's an error, I decided to just use the parser instead of the dict.
https://github.com/ska-sa/tango-simlib/blob/c777ada93cd4befecb4a8296f55d60f14d018a92/tango_simlib/model.py#L210-L213